inductiva/resources/resources.py

```diff
@@ -33,11 +37,25 @@ def _machine_group_list_to_str(machine_groups) -> str:
 
 
 def list_active_machine_groups():
```

Comment:
you can just call it `list_machine_groups` (consistent with the `get_machine_groups`).
Potentially even better would be `inductiva.resources.machine_groups.list()`  and `inductiva.resources.machine_groups.get()` which would be more consistent with the tasks...

Wdyt @begaiym-k and @luispcunha ?